deal.II-toolchain/packages/assimp.package

```diff
@@ -1,37 +1,59 @@
-VERSION=4.1.0
-CHECKSUM=83b53a10c38d964bd1e69da0606e2727
-#VERSION=3.3.1
-#CHECKSUM=fc57b024e80ebb13301bd0983826cad3
-NAME=v${VERSION}
-EXTRACTSTO=assimp-${VERSION}
-SOURCE=https://github.com/assimp/assimp/archive/
-PACKING=.tar.gz
+################################################################################
+## ASSIMP                                                                     ##
+################################################################################
+
+# In general, we recommend version 4.1.0 or higher.
```

Comment:
why?

Reply:
Because I was not able to compile assimp 3.3.1 at all neither on my local machine (WSL2 Ubuntu 20.04) nor on any github actions runners.